### ISSUE TITLE
Context menu?

### ISSUE THREAD

**Issue by braco** [tuvistavie/atomic-chrome#2]
Why not put the activation icon in the context menu on textareas? (right-click, ctrl-click, etc)

Could eliminate the need for the toolbar icon, which seems out of place.

**Comment by danhper** [comment]
I use a shortcut to activate the extension.
I agree the toolbar icon is not really useful,
but I think in this case, it would be good to have a default shortcut.
Do you think that a context menu could be more useful than a simple shortcut?

**Comment by braco** [comment]
I think so, it would target the element and seems obvious that you're performing an action there.

And maybe some action could be performed if the element loses focus?

**Comment by danhper** [comment]
I see. I agree this use of the contextual menu seems more natural.
I am going to change this, thank you for the feedback.

I would however rather not do anything when the element loses the focus,
as for example in Github, one would press the preview button and
make some modifications afterwards, so I think it could interfere
with a normal workflow.

**Comment by braco** [comment]
Good point! Maybe the DOM location could be stored and focus could be recaptured when the user begins typing in Atom again?
